src/app/InteractionModelDelegate.h

```diff
@@ -75,6 +76,49 @@ class InteractionModelDelegate
      */
     virtual CHIP_ERROR ReportError(const ReadClient * apReadClient, CHIP_ERROR aError) { return CHIP_ERROR_NOT_IMPLEMENTED; }
 
+    /**
+     * Notification that a Command Send has received an Invoke Command Response containing a status code.
+     * @param[in]  apCommandSender A current command sender which can identify the command sender to the consumer, particularly
+     * during multiple command interactions
+     * @param[in]  aGeneralCode   Status code defined by the standard
+     * @param[in]  aProtocolId    Protocol Id
```

Comment:
can we delete the parameter documentation that do not contain useful info (like protocol/endpoint/cluster/command id)?

Or does doxygen scream at us if we document a parameter but not another?

As a general style note: a method with 8 parameters is extreamly worrysome. Maybe we should encapsulate a command path (protocol id/code, cluster id, command id, index) as a single object to make intent clearer and argument count smaller.

Reply:
sure, would address it using #5624